app/assets/javascripts/active_admin/application.js.coffee

```diff
@@ -1,10 +1,15 @@
 # Initializers
 $(document).on 'ready page:load', ->
   # jQuery datepickers (also evaluates dynamically added HTML)
-  $(document).on 'focus', '.datepicker:not(.hasDatepicker)', ->
+  $(document).on 'focus', 'input.datepicker:not(.hasDatepicker)', ->
+    $input = $(@)
```

Comment:
@timoschilling I always prefix jQuery objects with a $. It's a personal preference, but helps remind the programmer that

1.) They don't have to re-initialize the object as jQuery.
2.) All of the jQuery object methods are available to that object at any time.

I find that it's a worthwhile distinction, but as I said before it's a personal preference. I'm fine with removing it.